app/templates/render-tree.hbs

```diff
@@ -1,3 +1,7 @@
+{{#unless isClosed}}
+  {{warning-message storage=storage}}
```

Comment:
Sorry, I slightly misspoke. You'd pass an action that sets `isClosed` instead. The component doesn't actually need to know anything, as the controller holds the value and the conditional